src/BackpackManager.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Backpack\CRUD;
+
+use Backpack\CRUD\app\Http\Controllers\Contracts\CrudControllerContract;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+
+final class BackpackManager
+{
+    private array $cruds = [];
+
+    private CrudPanel $crudPanelInstance;
+
+    private $requestController = null;
+
+    public function __construct()
+    {
+        $this->crudPanelInstance = new CrudPanel();
+    }
+
+    public function getCrudPanelInstance(): CrudPanel
+    {
+        return $this->crudPanelInstance;
+    }
+
+    public function crud(CrudControllerContract $controller): CrudPanel
+    {
+        $controllerClass = get_class($controller);
+
+        $this->requestController = $controllerClass;
+
+        if (isset($this->cruds[$controllerClass])) {
+            return $this->cruds[$controllerClass];
+        }
+
+        $instance = new CrudPanel();
+
+        $this->cruds[$controllerClass] = $instance;
+
+        return $this->cruds[$controllerClass];
+    }
+
+    public function crudFromController(string $controller): CrudPanel
+    {
+        $controller = new $controller();
+
+        $crud = $this->crud($controller);
+
+        $crud->setOperation('list');
```

Comment:
🚩 why hard-coded for the list operatin? In the future, we should be able to use BackpackManager to load a Form component, right?

Reply:
Maybe....

```php
getOperationFromController(string $controller, string $operation)
```

Or maybe we move this line to a separate method?! 🤷‍♂️